sentry/migrations/20190806011140_initial-tables/seed/up.sql

```diff
@@ -11,3 +11,74 @@ VALUES
705E6cB91632E7248E1B8f":"253890000000000000","0xa2F804EFcA463f8b28be16FF4A3bE975b979Dc3F":"4471440000000000000","0x57265f3b5dD6EE5E6C237EC126Ef0bd7Bdb6cB0a":"49290000000000000","0xE63f8A71758b58381D6D8Bf2678BA1A0EbC154B9":"207855000000000000","0xBFDC738DB9468cE1B98D19274855ec6e363B2249":"176514000000000000","0x18975F654d1035E4C2EC2F5a5047aa4DB27c015b":"405759000000000000","0x29020B170E8A15d345413a0e439C45b78A69d4E3":"310620000000000000","0x290defA5e7329dE9569b4C8e7556f56CD9564C73":"77190000000000000","0xEA4FCaa43dcB8dA109807F02486118702D1403E3":"58590000000000000","0x779ccF597Bd95655793024bf5B28E6dc28527635":"52638000000000000","0x7bf26452A91857Fb1334414D8F0Ea1F900Cf44dd":"101835000000000000","0xC311e00c0ecAd9444D29B74459A8A74C12d3973c":"9300000000000000","0xb07AB1636A3Cc38eBe1058AaFD0492cd3a9cC452":"123690000000000000","0xc57aaBADCFee078c9cc8DA7329E91818c12C5308":"44640000000000000","0x362c8F874cbccDf9081bc29443a538B54334F7c5":"7905000000000000","0xD34fcDEa923D0Aa10df10060a7F4dCCDdbe172f5":"281790000000000000","0x2A42596a55d6F7591790F590Fdbf7D2f2Cdb47F4":"60450000000000000","0x951acECCA3863FEbb8e40E5E68cB480Fa76bf161":"2325000000000000","0x3e76bc5Af5C6aA9CDEF2869fc22607afD2a0EbB9":"18600000000000000","0xac62B692572dDA8F96df7A686859037F3d479904":"37200000000000000","0x9d1d83D8beD5523fa1635eC6fddd2b9882258aF7":"15810000000000000","0xc63C197FF228C186A2eD3e5e378fce29dF4Db53A":"14973000000000000","0x980Cd24CB1692dEd82456225f2757953d743754D":"3813000000000000","0x5F8f70c8d10c2945ea2833F0Df22eBA32AcFa568":"930000000000000","0xC356E3693aeA749242073DE245790171a3DBe098":"930000000000000","0xE1E92e283D29e52750350c19021b641Fc5D444ac":"3720000000000000","0x283246511a9e60a5A937B4fb4250efB140894442":"3906000000000000","0x1232022f842363b9865F7b88a9aC654677bcbe89":"1860000000000000","0xB6F408aD7b8e0C433C6A6dc57D434391930B8A02":"1860000000000000","0x117480AB9DE66775F8EF5Ab54dBD0058282d6136":"930000000000000","0x0106a9dA9578760C1854f8666BAF07605452CD6b":"930000000000000","0xa7357939f51CEdBfAf0ce23d541C257905da86F4":"930000000000000","0x6353E3309B79dc203246Fa7E484CFaA0fA1BbB5B":"930000000000000","0x93753d542388746899ECF81fF14888dfAf8793dD":"22413000000000000","0x5d0AC46A1999d8F27B6fAd2b4AE110b599e053FD":"930000000000000","0x58DEB55AaAe43d2d91C8E84eCbCEb796Da223Ec0":"2790000000000000","0xe5C888a9d8bc5d7fE1b98f42af042a98acd35c83":"1860000000000000","0x59112662669a3e9814bE4238996929C593C279D4":"1023000000000000","0x9eb184C0dF6A0b9C43107c4CD8e29185E558945A":"1860000000000000","0x4d6F51e9D7A3966cd28748584c37A5d5aca4aa68":"930000000000000","0xB5f1AB60EA51B918D0eBB23f5f2f4A1972559D92":"930000000000000","0xD0ca406a411FAA6F00F75a16617af27CbE778343":"930000000000000","0xe3C19038238De9bcc3E735ec4968eCd45e04c837":"8400000000000000000"},"stateRoot":"bead8616d10aa90fe562a95a610173ccdf8c766edcf1c0ec0be89aba6467770c","signature":"0xece04ee2cfb42ac1582ba3728c60bc3252bd371d735e3a97a84c74985cd4dd96090cc172c1caa7b9229903d4f4366bdd26945532cb2f0d74dc8a2b63f43e38d11b","exhausted":true}',
```

Comment:
This doesn't seem to be right. This `NewState` has way to many balances and addresses + it says it's exhausted.